content/en/docs/tasks/debug-application-cluster/debug-cluster.md

```diff
@@ -29,6 +29,13 @@ kubectl get nodes
 
 And verify that all of the nodes you expect to see are present and that they are all in the `Ready` state.
 
+Also, you can run below command to get detailed information of overall cluster health.
+
+```shell
+
```

Comment:
This blank line doesn't seem to be needed here.

Reply:
Agree @sftim

Reply:
@rikatz you forgot to address this review comment!

Reply:
I think you mean @rkatti :)

Reply:
@sftim @makoscafee have addressed the comments. Pls, review.